packages/core/content-type-builder/server/src/controllers/content-types.ts

```diff
@@ -113,6 +117,29 @@ export default {
 
       const contentTypeService = getService('content-types');
 
+      // Find attributes with a `renamed` field that contains the original name
```

Comment:
as mentioned, this isn't how the final version will be implemented; we will probably send it as a separate object rather than tacking it on to the schema itself